### DISCUSSION TITLE
Provision Nginx / UFW / Redis

### DISCUSSION THREAD

**Discussion by jasperf** [deployphp/deployer#3501] [General]
Seems the provisioning scripts work with Caddy, some SQL databases, PHP. But there is no UFW firewall, Nginx and no Redis available yet. That is a setup a lot of Laravel sites and general PHP sites use. LEMP stacks with Redis and or some supervisors/workers. Like Ploi.io or Laravel Forge. Are there plans for adding provision recipes for Nginx that work with PHP FPM, Redis and for adding supervisor/workers?

**Comment by jasperf** [comment]
Never mind. Seems I asked this before.

**Comment by antonmedv** [comment]
Deployer provisions Redis and UFW too. Instead of Nginx, we use Caddy.

**Reply by jasperf** [reply]
Thanks. Good to hear Redis and UFW are there too. Might have to try with Caddy. As for a queue worker/horizon and supervisor, would have to look into adding a those with ease as well.

**Reply by antonmedv** [reply]
I have plans to add it too. Need to find some free time to implement it.
